bakerydemo/templates/base/include/header-blog.html

```diff
@@ -0,0 +1,23 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% if page.image %}
```

Comment:
So no image no header?

Reply:
Fixed this so blogs can show without hero images 👍